docs/tests/system_checklist.rst

```diff
@@ -0,0 +1,212 @@
+======================
+Socorro Test Checklist
+======================
+
+This is a high-level system-wide checklist for making sure Socorro is working
+correctly in a specific environment. It's a helpful template for figuring out
+what you need to change if you're pushing out a significant change.
+
+**Note:** This is used infrequently, so if you're about to make a significant change,
+you should go through the checklist to make sure the checklist is correct and
+that everything is working as expected and fix anything that's wrong, THEN
+make your change, then go through the checklist again.
+
+Lonnen the bear says, "Only you can prevent production fires!"
+
+Last updated: November 15th, 2017
+
+
+How to use
+==========
+
+"Significant change" can mean any number of things, so this is just a template.
+You should do the following:
+
+1. Copy and paste the contents of this into a Google doc.
```

Comment:
Is a google doc still the preferred method for this vs adding it as a comment to a PR or something?

Reply:
I don't think working through a system checklist is viable in Github PRs or Bugzilla. I think we're looking at Google Docs or Etherpad as better systems for working through the checklist and the non-trivial status bookkeeping.

Having said that, I'll change this line. I wouldn't use GitHub PRs or Bugzilla comments, but if someone else wants to do that, then power to them.